File: fizzbuzz.cpp
#include <iostream>

int main(int argc, char* argv[]) {
  int rng_start;
  int rng_end;
  std::cout << "Enter the start and end of the range: ";
  std::cin >> rng_start >> rng_end;
  if (rng_start >= rng_end) {
    std::cout << "The start must be less than the end." << std::endl;
    return 1;
  }

  for (int i = rng_start; i <= rng_end; i++) {
    if (i % 15 == 0) {
      std::cout << "FizzBuzz" << std::endl;
    } else if (i % 3 == 0) {
      std::cout << "Fizz" << std::endl;
    } else if (i % 5 == 0) {
      std::cout << "Buzz" << std::endl;
    } else {
      std::cout << i << std::endl;
    }
  }

  return 0;
}

File: fizzbuzz.py
#!/usr/bin/env python

import sys

rng_start = input('Enter the starting number: ')
rng_end = input('Enter the ending number: ')
try:
    rng_start = int(rng_start)
    rng_end = int(rng_end)
except ValueError:
    print('The range values must be pure integers.')
    sys.exit(1)
if rng_start >= rng_end:
    print('The starting number must be less than the ending number.')
    sys.exit(1)

for i in range(rng_start, rng_end+1):
    if i % 15 == 0:
        print('FizzBuzz')
    elif i % 3 == 0:
        print('Fizz')
    elif i % 5 == 0:
        print('Buzz')
    else:
        print(i)
